scripts/variantstore/wdl/GvsJointVariantCalling.wdl

```diff
@@ -0,0 +1,88 @@
+version 1.0
+
+import "GvsUnified.wdl" as GvsUnified
+
+workflow GvsJointVariantCalling {
+    input {
+        String dataset_name
+        String project_id
+        Array[String] external_sample_names
+        Array[File] input_vcfs
+        Array[File] input_vcf_indexes
+        String callset_identifier
```

Comment:
nit: I believe elsewhere we are using "call_set_identifier"

Reply:
oh god I hate that, but I know that's how it is in the database

silly Q---is it the same thing?!??! Here it's being used as the string for the filter_set_name, the prepare table and the vcf prefix.